tests/test_cloudformation/test_stack_parsing.py

```diff
@@ -78,7 +78,10 @@
 ssm_parameter = {
     "Parameters": {
         "SingleParamCfn": {"Type": "AWS::SSM::Parameter::Value<String>"},
-        "ListParamCfn": {"Type": "AWS::SSM::Parameter::Value<List<String>>"},
+        "ListParamCfn": {
+            "Type": "AWS::SSM::Parameter::Value<List<String>>",
+            "Default": "/path/to/list/param",
```

Comment:
We'll have to verify that this value is actually retrieved from SSM, i.e. extend the test to add something to the parameter store, and assert afterwards that the values match up

Reply:
That is being done here: https://github.com/sahilshah6196/moto/blob/master/tests/test_cloudformation/test_stack_parsing.py#L528
The `ssm_parameter` on line 81 is being used in the test I linked above. 
Sorry if I am misunderstanding the request.

Reply:
That test only verifies the user-defined value, as that is passed in as part of `FakeStack(parameters=..)`
In order to assert that the default works, we should also create a FakeStack without passing in a value:
```
stack = FakeStack(
            stack_id="test_id",
            name="test_stack",
            template=ssm_parameter_template_json,
            parameters={
                "SingleParamCfn": "/path/to/single/param" # Not passing in a value for ListParamCfn!
            },
            region_name="us-west-1",
        )
       # and copy the same asserts, to verify that the 'ListParamCfn' now has the default value
```

Reply:
Extended that test: 3486f7a
@bblommers That looks good?

Reply:
Got an unwanted change in there, sorry. Removed: 8e65e26

Reply:
Yes, that's exactly what I had in mind